core/src/autogluon/core/features/types.py

```diff
@@ -31,6 +31,9 @@
 # feature is a generated feature based off of a text feature that is an ngram.
 S_TEXT_NGRAM = 'text_ngram'
 
+# feature is an object type that contains a string url to an image that can be utilized in computer vision
+S_IMAGE_URL = 'image_url'
```

Comment:
Might _path be more appropriate suffix than _url?

Reply:
`image_url` is missing from list of possible special types in FeatureMetadata documentation

Reply:
Niche use-case but may want to clarify:  how user can leverage both image and its file-path both as predictive features

Reply:
renamed to `image_path`. Regarding using path as both text and image feature at the same time: Let's leave that for later, seems really niche.

Reply:
added `image_path` to FeatureMetadata docs